script.skinshortcuts/default.py

```diff
@@ -0,0 +1,397 @@
+# coding=utf-8
```

Comment:
It's a good practice to keep this file as small as possible as it's the only file not compiled to bytecode (as it's the entrypoint). Please move the codebase to `/resources/lib/` and in this file just do something like:

```python
import sys
from resources.lib import myaddon

myaddon.run(sys.argv)
```